.vib/influxdb/goss/influxdb.yaml

```diff
@@ -0,0 +1,6 @@
+command:
+  check-influxd:
+    exec: timeout 5s influxd || true
```

Comment:
Checking this command locally, `influxdb` fails due to a permission error. In this case, I'm not sure this test has any value besides checking the binary runs.

```
$ influxd
2023-03-15T18:10:26.634468Z	info	Welcome to InfluxDB	{"log_id": "0gacIxnW000", "version": "v2.6.1", "commit": "9dcf880", "build_date": "2023-03-15T18:10:26Z", "log_level": "info"}
2023-03-15T18:10:26.634738Z	error	Failed opening bolt	{"log_id": "0gacIxnW000", "error": "unable to create directory /.influxdbv2/influxd.bolt: mkdir /.influxdbv2: permission denied"}
Error: unable to create directory /.influxdbv2/influxd.bolt: mkdir /.influxdbv2: permission denied
See 'influxd -h' for help
```

Additionally, the command itself seems a bit hacky and can give a false perception of the influx DB running without issues (and of you having to timeout it to finish the test).